Source/BugsnagClient.m

```diff
@@ -269,6 +275,7 @@ - (id)initWithConfiguration:(BugsnagConfiguration *)initConfiguration {
         [self metadataChanged:self.configuration.metadata];
         [self metadataChanged:self.configuration.config];
         [self metadataChanged:self.state];
+        _pluginClient = [[BugsnagPluginClient alloc] initWithPlugins:self.configuration.plugins];
```

Comment:
Any reason we shouldn't use `[self pluginClient...` to keep this consistent with other properties such as `SessionTracker` which have similar visibility/usage.

Reply:
I'll defer to @robinmacharg on this one as I believe there's a lot of nuance and differing opinion on when `self` should be used vs accessing the property directly, and it's something we should standardise on.

Reply:
I guess I was looking from a consistency point of view - this seems to be different to how we're accessing other properties that aren't intended for public use (e.g. SessionTracker)

Reply:
I'll update this to be consistent for now, then we can always discuss whether to use `self` or direct field access later on.

Reply:
The short of it is that using `_property` accesses the property directly. Using `self` uses synthetic getters/setters that we could, in theory, override in the future if we needed to.  ObjC gives us the getters/setters for free, unseen.  There's potentially a minuscule performance hit but opinion seems to be that this is probably optimised out and so `self.property` is preferable.